include/qpid/dispatch/server.h

```diff
@@ -418,6 +418,22 @@ typedef struct qd_server_config_t {
     uint32_t max_frame_size;
 
     /**
+     * The max_sessions value is the number of sessions allowed on the Connection. 
+     * This value minus one is the Open performative channel-max setting.
+     */
+    uint32_t max_sessions;
```

Comment:
There should be a comment that says if you specify zero then you get the default number. Using channel_max is awkward in its own way: if the setting is 1 then you get 2.
There is an identical setting in policy that already used the 1..N numbering (and not 0..N-1). They ought to be the same either way.